dexs/ashswap/index.ts

```diff
@@ -14,18 +20,31 @@ const VolumeQuery = `
 `
 
 const fetch = async (timestamp: number) => {
-  const results = await request(API_URL, VolumeQuery);
+  const dayTimestamp = getUniqStartOfTodayTimestamp(new Date(timestamp * 1000))
+  const results: any[] = (await fetchURL(API_URL)).data;
+  const volumes = results.map((e: any) => {
+    const [time, volume] = e;
+    return {
+      volume,
+      time
+    }
+  });
+  const dailyVolume = volumes.find((e: IVolume) => e.time === dayTimestamp);
+  const totalVolume = volumes
+    .filter((e: IVolume) => e.time <= dayTimestamp)
+    .reduce((a: number, e: IVolume) => a+e.volume, 0);
   return {
-    dailyVolume: results.defillama.totalVolumeUSD24h,
-    timestamp: getUniqStartOfTodayTimestamp(new Date(timestamp * 1000)),
+    dailyVolume: dailyVolume ? `${totalVolume}` : undefined,
```

Comment:
@dtmkeng hi sir, it can be mistake: the fallback of `dailyVolume` is `totalVolume`.
And our graphql service is wrapper of this api, it has more caching and more stable than this api, so please use our graphql service instead of this change.